src/GovUk.Education.ExploreEducationStatistics.Data.Services/ViewModels/Meta/ResultSubjectMetaViewModel.cs

```diff
@@ -1,27 +1,32 @@
+#nullable enable
 using System.Collections.Generic;
-using GovUk.Education.ExploreEducationStatistics.Data.Model;
 
 namespace GovUk.Education.ExploreEducationStatistics.Data.Services.ViewModels.Meta
 {
-    public class ResultSubjectMetaViewModel
+    public record ResultSubjectMetaViewModel
     {
-        public Dictionary<string, FilterMetaViewModel> Filters { get; set; }
+        public Dictionary<string, FilterMetaViewModel> Filters { get; init; } = new();
 
-        public IEnumerable<FootnoteViewModel> Footnotes { get; set; }
+        public List<FootnoteViewModel> Footnotes { get; init; } = new();
 
-        public IEnumerable<IndicatorMetaViewModel> Indicators { get; set; }
+        public List<IndicatorMetaViewModel> Indicators { get; init; } = new();
 
-        // TODO SOW8 EES-2881 Change this to support both proposed and old location formats 
-        public IEnumerable<ObservationalUnitMetaViewModel> Locations { get; set; }
+        // TODO EES-2917 Work out how to maintain backwards compatibility for Locations with the hierarchical response type
```

Comment:
Also in terms of backwards-compatibility, we might need to invalidate all of the cached "/cache/publication/{publication-slug}/release/{release-slug}/subjects", "/cache/publication/{publication-slug}/release/{release-slug}/subject-meta" and "/cache/publication/{publication-slug}/release/{release-slug}/fast-track-results" folders fro the public site with a manual endpoint when deploying the work that enables the hierarchical feature.

If we're maintaining backwards-compatibility in existing fast tracks and subjects, we can probably leave them be.

Be good to discuss.